packages/block-library/src/table/edit.js

```diff
@@ -0,0 +1,427 @@
+/**
+ * External dependencies
+ */
+
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+
+import { Fragment, Component } from '@wordpress/element';
+import { InspectorControls, BlockControls, RichText } from '@wordpress/editor';
+import { __ } from '@wordpress/i18n';
+import {
+	PanelBody,
+	ToggleControl,
+	TextControl,
+	Button,
+	Toolbar,
+	DropdownMenu,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+
+import {
+	createTable,
+	updateCellContent,
+	insertRow,
+	deleteRow,
+	insertColumn,
+	deleteColumn,
+} from './state';
+
+/**
+ * Browser dependencies
+ */
+
+const { parseInt } = window;
```

Comment:
I don't think this is a browser feature, it is part of the language:

![screen shot 2018-08-23 at 08 14 10](https://user-images.githubusercontent.com/699132/44507898-8de54a00-a6ac-11e8-9734-22dba7cb0b86.png)